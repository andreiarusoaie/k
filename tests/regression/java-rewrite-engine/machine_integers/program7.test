 MyListItem(uvalueMInt(miMInt(saddMInt(mi(31, smaxMInt(31)), mi(31, 1))))),
 MyListItem(overflowMInt(saddMInt(mi(31, smaxMInt(31)), mi(31, 1))))

