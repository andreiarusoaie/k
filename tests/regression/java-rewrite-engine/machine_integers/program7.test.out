<k>
    MyListItem ( 1073741824 ) , MyListItem ( true )
</k>
